doc/api/events.md

```diff
@@ -319,7 +319,7 @@ console.log(util.inspect(server.listeners('connection')));
 
 ### emitter.on(eventName, listener)
 
-* `eventName` {string|Symbol} The name of the event.
+* `eventName` {String|Symbol} The name of the event.
```

Comment:
I'm not sure I agree with this kind of change. I know @jasnell had mentioned some time back that he was planning on going through the docs and making these types and their capitalization consistent.